conductr_cli/sandbox_stop.py

```diff
@@ -1,8 +1,16 @@
-from conductr_cli import sandbox_stop_docker
+from conductr_cli import sandbox_stop_docker, sandbox_stop_jvm
+from conductr_cli.sandbox_common import major_version, RUNNING_CONDUCTR_VERSION_FILE_PATH
 
 
 def stop(args):
     """`sandbox stop` command"""
 
-    sandbox_stop_docker.stop(args)
+    if not hasattr(args, 'image_version'):
+        with open(RUNNING_CONDUCTR_VERSION_FILE_PATH, 'r') as file:
+            args.image_version = file.read()
+
+    if args.image_version == 1:
+        sandbox_stop_docker.stop(args)
+    elif args.image_version != 1:
+        sandbox_stop_jvm.stop(args)
```

Comment:
If the `sandbox_stop_docker.stop` function is called and docker is not running then we currently throw this error:

```
Error: Encountered unexpected error.
Error: Reason: CalledProcessError Command '['docker', 'ps', '--all', '--quiet', '--filter', 'name=cond-']' returned non-zero exit status 1.
Error: Further information of the error can be found in the error log file: /Users/mj/.conductr/errors.log
```